Addition of Terms of Service + Privacy Policy (#2)
PrimerNetwork/anitra-docs: src/terms.md

# Terms of Service and Privacy Policy

Last Update and Effective: December 28, 2022

> These Terms of Serivce and Privacy Policy applies to the Discord Bot: Anitra#9270 (ID: 1043892494280175616)

Reviewer: ```suggestion
This Terms of Serivce and Privacy Policy applies to the Discord Bot: Anitra#9270 (ID: 1043892494280175616)
```

## Terms of Service

### Usage Agreement

> By inviting the bot and using it's features (including but not limited to: commands, suggestion system, button roles), you agree to the below Terms of Service and Privacy Policy (the "Policy") of the bot.

Reviewer: ```suggestion
By inviting the bot and using its features (including but not limited to: commands, suggestion system, button roles), you agree to the below Terms of Service and Privacy Policy (the "Policy") of the bot.
```

You acknowledge that you have the right to freely use the bot on any Discord Server you share with the bot, you have the right to invite it to a server that you have "Manage Server" rights for and that this right may be revoked at any point in time with or without reason to be determined by PrimerNetwork. 

Inviting the bot may collect certain and specific data as defined in the [Policy](./terms.md).
The intended usage of this data is for the core functions of the bot including but not limited to: command handling, guild specific settings, the Suggestions System, Button Roles, Private Rooms and Automatically Assigned Roles.

### Intended Age

> Anitra may not be used by any person or persons under the specified minimal age defined in Discord's [Terms of Service](https://discord.com/terms). Useage of this bot while under this speicified age will result in a voliation of these terms and will result in blacklisting from all PrimerNetwork Bots as well as removal and blacklists from any and all servers owned by the person or persons.

Reviewer: ```suggestion
Anitra may not be used by any person or persons under the specified minimal age defined in Discord's [Terms of Service](https://discord.com/terms). Usage of this bot while under this specified age will result in a violation of these terms and will result in blacklisting from all PrimerNetwork Bots as well as removal and blacklists from any and all servers owned by the person or persons.
```

### Affiliation

> Anitra is not affiliated with, supported or made by Discord In.c

Reviewer: ```suggestion
Anitra is not affiliated with, supported or made by Discord Inc.
```

> Any direct connection with Discord or any of it's Trademark objects is purely coincidental. PrimerNetwork does not claim to have the copyright ownership of any and all Discord Assets, trademarks or intellectual property.

Reviewer: ```suggestion
Any direct connection with Discord or any of its Trademark objects is purely coincidental. PrimerNetwork does not claim to have the copyright ownership of any and all Discord Assets, trademarks or intellectual property.
```

### Liability

> PrimerNetwork may not be made liable for individuals breaking these Terms at any given time. PrimerNetwork has faith in the person or persons using the bot to be truthful about their information and missusing the bot or the Services profided by Discord in any malicious way.

Reviewer: ```suggestion
PrimerNetwork may not be made liable for individuals breaking these Terms at any given time. PrimerNetwork has faith in the person or persons using the bot to be truthful about their information and misusing the bot or the Services provided by Discord in any malicious way.
```

### Updates to the Policy and Rights of PrimerNetwork

> PrimerNetwork reserves the right to update these terms at our discretion, giving you a 1 (one) week (7 days) period to opt out of these terms if you do not agree with the changes. 

Reviewer: ```suggestion
PrimerNetwork reserves the right to update these terms at our discretion, giving you a 1 (one) week (7 days) period to opt out of these terms if you do not agree with the changes.
```

You may opt out by removing Anitra from any server you have the rights for.

## Privacy Policy

> ### Data Anitra Collects and Usage of the Data

Reviewer: ```suggestion
### Data Collection and Usage of Data
```

Anitra collects data to maintain the features provided in the bot's service. Below is a list of the data we collect and how it is used:

- Guild/Server IDs:
    - The Guild/Server ID is used to maintain and use the settings and features users enable within the server. This includes features like Button Roles, Auto Roles, Private Rooms and Suggestions as well as our error logging.
- Channel IDs:

>     - Channel IDs are used to send data within a speicific channel. For suggestions, any suggestion made by a user is sent to the speicified channel. For button roles, this is to store the channel that the button roles message is tied to. For logging purposes (such as error logs), this data is used to tell us more about the error in occured.

Reviewer: ```suggestion
    - Channel IDs are used to send data within a specific channel. For suggestions, any suggestion made by a user is sent to the specified channel. For button roles, this is to store the channel that the button roles message is tied to. For logging purposes (such as error logs), this data is used to tell us more about the error in occurred.
```

- User IDs:

>     - Anitra uses User IDs to ensure the bot is able to function properly and not get confused with what user is preforming what action.

Reviewer: ```suggestion
    - Anitra uses User IDs to ensure the bot is able to function properly and not get confused with what user is performing what action.
```
